docs/conf.py

```diff
@@ -137,7 +137,7 @@ def data_dictionary_metadata_to_rst(app):
     """Export data dictionary metadata to RST for inclusion in the documentation."""
     # Create an RST Data Dictionary for the PUDL DB:
     print("Exporting PUDL DB data dictionary metadata to RST.")
-    skip_names = ["datasets", "accumulated_depreciation_ferc1", "entity_types_eia"]
```

Comment:
The never-quite-finished `entity_types_eia` table (now `core_eia__codes_entity_types`) was malingering and creating weird special cases so I excised it.